plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/context/ui/SingleLifeline.xwt

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Composite xmlns="http://www.eclipse.org/xwt/presentation"
+	xmlns:ppe="clr-namespace:org.eclipse.papyrus.infra.properties.ui.widgets"
+	xmlns:ppel="clr-namespace:org.eclipse.papyrus.infra.properties.ui.widgets.layout"
+	xmlns:uml="clr-namespace:org.eclipse.papyrus.uml.properties.widgets"
+	xmlns:x="http://www.eclipse.org/xwt">
+	<Composite.layout>
+		<ppel:PropertiesLayout></ppel:PropertiesLayout>
+	</Composite.layout>
+	<Composite>
+		<Composite.layout>
+			<ppel:PropertiesLayout numColumns="2"></ppel:PropertiesLayout>
+		</Composite.layout>
+		<ppe:StringEditor input="{Binding}" property="LWSequenceDiagram:Lifeline:name"></ppe:StringEditor>
+		<ppe:StringEditor input="{Binding}" property="LWSequenceDiagram:Lifeline:label"></ppe:StringEditor>
```

Comment:
The `label` property isn't defined in the data-context.  Likewise the `SingleMessage.xwt`.

Reply:
I don't really know how the internationalization property works. The label property isn't defined in the UML.ctx either, nevertheless, it is used in the xwts. I did it analogously to the UML.ctx. Do you know whether we have to define it? I suspect it is sort of an aspect that get's weaved in by some other Papyrus internationalization plug-in.

Reply:
That's odd.  I don't know how the i18n framework works, either.  If it seems to function properly at run-time, then I suppose it's best to follow the Papyrus practice.  Have you tested it in your new properties tab?  I'm wary of a thing that works that doesn't look like it should.